kore/test/Test/Kore/Step/SMT/Representation/Builders.hs

```diff
@@ -138,8 +138,9 @@ unresolvedConstructorSymbol identifier resultSort argumentSorts =
         , declaration = AST.SymbolConstructor
             AST.IndirectSymbolDeclaration
                 { name = encodable
-                , resultSort = AST.SortReference resultSort
-                , argumentSorts = map AST.SortReference argumentSorts
+                , sortDependencies =
+                    AST.SortReference resultSort
+                    : map AST.SortReference argumentSorts
```

Comment:
```suggestion
                    AST.SortReference <$> resultSort : argumentSorts
```

Reply:
There are a few more places where this change could be applied if you agree with it